configs/isanet/isanet_r50-d8_512x512_80k_ade20k.py

```diff
@@ -0,0 +1,7 @@
+_base_ = [
+    '../_base_/models/isanet_r50-d8.py', '../_base_/datasets/ade20k.py',
+    '../_base_/default_runtime.py', '../_base_/schedules/schedule_80k.py'
+]
+model = dict(
+    decode_head=dict(num_classes=150), auxiliary_head=dict(num_classes=150))
+test_cfg = dict(mode='whole')
```

Comment:
`train_cfg` and `test_cfg` have been moved to `model`, and` test_cfg = dict(mode='whole')` has already defined in `configs/_base_/models/isanet_r50-d8.py`, so this line is redundant.